src/machine/uart.go

```diff
@@ -1,11 +1,13 @@
-// +build avr nrf
+// +build avr nrf stm32
 
 package machine
 
 import "errors"
 
 type UARTConfig struct {
 	BaudRate uint32
+	TXPin    uint8
+	RXPin    uint8
```

Comment:
For I2C and SPI, the -Pin suffix is not used, they simply use `MOSI`, `SDA`, etc. It would be nice to keep the naming of these pins consistent.

Reply:
OK I will make that change.